src/Microsoft.Azure.WebJobs.Host/Executors/TriggeredFunctionExecutor.cs

```diff
@@ -51,10 +55,18 @@ public async Task<FunctionResult> TryExecuteAsync(TriggeredFunctionData input, C
                 };
             }
 
+            // initial attempt
```

Comment:
For all 3 of these call sites, I'm wondering why the initial invocation is outside of the TryExecuteAsync extension method. That method should be able to handle everything, including the first invocation, so we don't have to repeat the code?